docs/integrations/sources/linnworks.md

```diff
@@ -1,54 +1,71 @@
 # Linnworks
 
-## Sync overview
+This page contains the setup guide and reference information for the [Linnworks](https://www.linnworks.com) source connector.
 
-Linnworks source supports both Full Refresh and Incremental syncs. You can choose if this connector will copy only the new or updated data, or all rows in the tables and columns you set up for replication, every time a sync is run.
+## Prerequisites
 
-Airbyte uses [Linnworks API](https://apps.linnworks.net/Api) to fetch data from Linnworks.
+- A Linnworks account
 
-### Output schema
+## Setup guide
 
-This Source is capable of syncing the following data as streams:
+### Generate Credentials in Linnworks
 
-- [StockLocations](https://apps.linnworks.net/Api/Method/Inventory-GetStockLocations)
-- [StockLocationDetails](https://apps.linnworks.net/Api/Method/Locations-GetLocation)
-- [StockItems](https://apps.linnworks.net//Api/Method/Stock-GetStockItemsFull)
-- [ProcessedOrders](https://apps.linnworks.net/Api/Method/ProcessedOrders-SearchProcessedOrders)
-- [ProcessedOrderDetails](https://apps.linnworks.net/Api/Method/Orders-GetOrdersById)
+1. The Linnworks platform has two portals: seller and developer. To generate the necessary credentials, log in to the [developer portal](https://developer.linnworks.com) and select **+ New App**.
```

Comment:
can you capitalize Seller and Developer where written here?

Reply:
Done!